### DISCUSSION TITLE
keybind of discord (shift + ')

### DISCUSSION THREAD

**Discussion by MTSKaioken** [flamewave000/discord-overlay#26] [Ideas]
## Feature

this utility inside the softwares

![imagem_2024-01-24_133948560](https://github.com/flamewave000/discord-overlay/assets/50028868/900d7358-695b-43ac-a7b1-83d0aad224be)

**Comment by flamewave000** [comment]
Hi, I'm sorry to say I'm not sure what your are looking for. Is it that you want to change the discord keybinding for ``Shift + ` ``? If so, I don't control the keybinding, so it would need to be changed inside of Discord itself.

You can do so here: `Settings -> Game Overlay -> Toggle Overlay Lock`

![image](https://github.com/flamewave000/discord-overlay/assets/3588046/9a645b5b-1375-45e4-925b-541a685a157a)

**Reply by MTSKaioken** [reply]
Sorry, I didn't give many details... but what I wanted to say is that it would be interesting to have access to this shortcut through your software. In fact, it works, but only when your software is in focus.

**Reply by flamewave000** [reply]
So unfortunately it would not be possible for me to provide that shortcut when other programs are focused. Windows has a messaging system under the hood and Discord is listening to the input for my program, but not the program you have in focus. I wish Discord provided an actual SDK for their overlay like Steam does, but unfortunately they use a **very** hacky approach to their implementation.
